guides/source/api_documentation_guidelines.md

```diff
@@ -283,7 +283,7 @@ In lists of options, parameters, etc. use a hyphen between the item and its desc
 # * <tt>:allow_nil</tt> - Skip validation if attribute is +nil+.
 ```
 
-The description starts in upper case and ends with a full stop-it's standard English.
+The description starts in upper case and ends with a full stop--it's standard English.
```

Comment:
Since this will be rendered in a proportional (not monospace) font, let's use a true em dash:

```suggestion
The description starts in upper case and ends with a full stop—it's standard English.
```